xml/Makefile.am

```diff
@@ -20,7 +20,10 @@ MAINTAINERCLEANFILES    = Makefile.in
 RNGdir			= $(CRM_SCHEMA_DIRECTORY)
 
 xsltdir			= $(RNGdir)
-xslt_DATA		= $(top_srcdir)/xml/upgrade-*.xsl
+dist_xslt_DATA		= $(top_srcdir)/xml/upgrade-1.3.xsl \
+			  $(top_srcdir)/xml/upgrade-2.10.xsl \
```

Comment:
not necessary to change, but upgrade-[0-9]*.xsl would catch them

Reply:
Not that it matters _now_ (`make dist` not being used for authoritative
distribution sources), but work-in-process transformations shouldn't be
shipped until explicitly approved (consequently, the upgrades won't
happen up to experimental RNG version).